### PR TITLE
AG-11385 Use aria-disabled="true" instead of disabled

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-11385

The guideline is to make buttons of toolbars/menus focusable even when they are disabled. This is so that the buttons are discoverable to screenreader users.

Reference:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/toolbar_role

### CODE REVIEW

**Review by lsjroberts — COMMENTED** [review]
If I understand this correctly - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

I think we'll need to add a check on the `onclick` for the button's `ariaDisabled === 'true'` to stop it from firing?

**Comment by olegat** [comment]
I've added `onclick` listeners to `aria-disabled="true"` buttons.

File: packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts [hunk 1]
```diff
@@ -276,10 +277,14 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
     }
 
     private createActionElement({ id, label, type, action }: ContextMenuAction): HTMLElement {
+        let button: HTMLElement;
         if (id && this.registry.isDisabled(id)) {
-            return this.createDisabledElement(label);
+            button = this.createDisabledElement(label);
+        } else {
+            button = this.createButtonElement(label);
```
Comment (lsjroberts) [inline packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts]: nit: could merge `createButtonElement` and `createDisabledElement` as they're so similar? Then add the `ariaDisabled` in here if disabled?

**Review by lsjroberts — APPROVED** [review]
LGTM

File: packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts (hunk 1 above)
Comment (olegat) [reply packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts]: Done.

**Pull request merged**